dockerfiles/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ros:noetic
+
+ADD http://robotpkg.openrobots.org/packages/debian/robotpkg.key /
+RUN --mount=type=cache,sharing=locked,target=/var/cache/apt \
+    --mount=type=cache,sharing=locked,target=/var/lib/apt \
+    apt-key add /robotpkg.key \
+ && echo "deb [arch=amd64] http://robotpkg.openrobots.org/packages/debian/pub focal robotpkg" > /etc/apt/sources.list.d/robotpkg.list \
+ && apt-get update -y \
+ && DEBIAN_FRONTEND=noninteractive apt-get install -qqy \
+    oxygen-icon-theme \
+    robotpkg-py38-hpp-environments \
+    robotpkg-py38-hpp-manipulation-corba \
+    robotpkg-py38-qt5-hpp-gepetto-viewer \
+    robotpkg-py38-qt5-hpp-gui \
+    robotpkg-py38-qt5-hpp-plot \
+    robotpkg-py38-hpp-tutorials \
```

Comment:
if you add robotpkg-py38-hpp-tutorials, you can remove robotpkg-py38-hpp-manipulation-corba, as it is a direct dependency

Reply:
ok :)